src/main/resources/plugin.yml

```diff
@@ -1,5 +1,5 @@
 name: ${name}
 version: ${version}
 main: com.github.soerxpso.xpspawners.XPSpawners
-
 depend: [CivModCore]
+softdepend: [Citadel, Bastion]
```

Comment:
This is just to make sure that "placement" event will be called after Citadel and Bastion.
Now all events - in XPSpawners, Citadel and Bastion have the same priority LOW.
Probably I should increase priority in XPSpawners to NORMAL or even HIGH to be even more sure that it will not register 'ghost' spawner?